dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -403,6 +403,66 @@ class ApplicationsControllerTest < ::ActionController::TestCase
       assert_equal expected_log, @csp_facilitator_application.sanitize_status_timestamp_change_log
     end
 
+    test 'update appends to timestamp log if workshop admin changes application from unlocked to locked' do
+      sign_in @workshop_admin
+      @csp_facilitator_application.update(status_timestamp_change_log: '[]', locked_at: nil)
+      @csp_facilitator_application.reload
+
+      assert_equal [], @csp_facilitator_application.sanitize_status_timestamp_change_log
+      assert_equal false, @csp_facilitator_application.locked?
```

Comment:
Do
`refute @csp_facilitator_application.locked?` - it's the same as `assert_equal false, <whatever>`

Reply:
Will do